pynecone/components/graphing/victory.py

```diff
@@ -0,0 +1,465 @@
+"""Component for displaying a plotly graph."""
+
+from typing import Any, Dict, Union, List, Optional
+
+from pynecone.components.component import Component
+from pynecone.components.tags import Tag
+from pynecone.var import Var
+
+
+def format_line(x: List, y: List) -> List:
+    """Format line data."""
+    data = []
+    if len(x) != len(y):
+        raise ValueError("x and y must be the same length")
+
+    for i in range(len(x)):
```

Comment:
This can be simplified to
```
return [{"x": x[i], "y": y[i]} for i in range(len(x))]
```
I think it may be worth it to move that line out to a helper function since it's used so much in this file, such that you can just call something like.
```
return create_data({"x": x, "y": y})
```
So you can just give it the dict of lists and it can convert it to a list of dicts.
It can even the the length assertions within the helper functions instead of repeating it.